en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/S/Scrying.md

```diff
@@ -8,19 +27,22 @@ _5th-level divination_
 **Components:** V, S, M (a focus worth at least 1,000 gp, such as a crystal ball, a silver mirror, or a font filled with holy water) \
 **Duration:** Concentration, up to 10 minutes
 
-You can observe a specific creature on the same plane of existence as you or a location you’ve observed before that you choose.
-The difficulty of the saving throw for this spell is modified by your knowledge of the target and whether you possess a physical item with a connection to the target.
+You attempt to observe a specific creature on the same plane of existence as yourself that you choose.
+The difficulty of the Wisdom saving throw for this spell is modified by your knowledge of the target and whether you possess a physical item with a connection to the target.
 
+On a successful save, the target isn’t affected, and you can’t use this spell against it again for 24 hours.
 On a failed save, you can see and hear the target through an invisible sensor that appears within 10 feet of it and moves with the target.
-Any creature who can see invisibility or rolls a critical success on its saving throw perceives the sensor as a fist-sized glowing orb hovering in the air. Creatures cannot sense you through the sensor.
+Any creature who can see invisibility or rolls a critical success on its saving throw perceives the sensor as a fist-sized glowing orb hovering in the air.
+Creatures cannot sense you through the sensor.
 
-If you choose to target a location, the sensor appears at that location and is immobile.
-
-| Knowledge Or Physical Connection                                 | DC Modifier |
+| Knowledge or Physical Connection                                 | DC Modifier |
 |:-----------------------------------------------------------------|:-----------:|
 | You have heard of the target                                     |      –5     |
 | You have met the target                                          |      +0     |
 | You know the target well                                         |      +5     |
 | Likeness                                                         |      +2     |
 | Possession                                                       |      +4     |
 | Lock of hair, nail clipping, tooth, finger, true name, and so on |     +10     |
+
+Instead of targeting a creature, you can choose a location you’ve observed before.
+If you choose to target a location, the sensor appears at that location and is immobile.
```

Comment:
```suggestion
Instead of targeting a creature, you can choose a location on the same plane of existence and that you’ve observed before.
If you choose to target a location, the sensor appears at that location and is immobile.
```